packages/ketcher-react/src/script/ui/dialog/toolbox/sgroup.module.less

```diff
@@ -14,48 +14,73 @@
  * limitations under the License.
  ***************************************************************************/
 @import "src/style/variables";
-
+@import "src/style/mixins";
+// stylelint-disable selector-pseudo-class-no-unknown
+// stylelint-disable no-descending-specificity
 .sgroup {
   width: @width-small;
+  border-radius: 6px;
+  background-color: @color-background-primary;
+
+  header {
```

Comment:
'header' styles should be moved to Dialog styles

Reply:
moved header to dialog